tests/HelperTest.php

```diff
@@ -42,7 +50,7 @@ public function testSubstitutesMarkup(string $string, array $replace, array $mar
         self::assertEquals($expected, $actual);
     }
 
-    public function markupSubstitutionProvider(): array
+    public static function markupSubstitutionProvider(): array
```

Comment:
OOC: Is this the new shiny recommendation for data providers?

Reply:
This is a PHPUnit 10 change